draft-jholland-quic-multicast.md

```diff
@@ -197,6 +197,9 @@ Since clients can join later than a channel began, it is RECOMMENDED that client
 Clients should therefore begin with a high initial_max_streams_uni or send an early MAX_STREAMS type 0x13 value (see Section 19.11 of {{RFC9000}}) with a high limit.
 Clients MAY use the maximum 2^60 for this high initial limit, but the specific choice is implementation-dependent.
 
+The same stream ID may be used in both one or more multicast channels and the unicast connection.
```

Comment:
I think this paragraph needs to refer to https://www.rfc-editor.org/rfc/rfc9000#section-2.2, and basically needs to restate what it says there and emphasize that this spec changes nothing about that handling, it only means there are extra ways that a client endpoint might receive data for a stream multiple times.  I don't think we have a new MUST here, we just have the same MUST from RFC 9000 section 2.2.